src/microbe_stage/systems/MicrobeRenderPrioritySystem.cs

```diff
@@ -59,12 +59,13 @@ protected override void Update(float state, in Entity entity)
 
         try
         {
-            // First material is always the membrane, which gets the base value. Organelles are rendered above it.
+            // First material is always the membrane, which gets the base value with an offset,
+            // so that organelles are rendered below it.
             var materialList = materials.Materials;
 
             if (materialList.Length > 0)
             {
-                materialList[0].RenderPriority = renderOrder.RenderPriority;
+                materialList[0].RenderPriority = renderOrder.RenderPriority + 20;
```

Comment:
I think this 20 should be in constants so that it is clear there's this kind of hardcoded value exists and might conflict with something if there are future code changes.

Reply:
Done